src/views/Edit.js

```diff
@@ -1,22 +1,87 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import { useParams} from "react-router";
+import { useHistory } from "react-router-dom";
+
+export default function Edit(props) {
+    let { id } = useParams();
+    let history = useHistory();
+
+    const [start, setStart] = useState('');
+    const [end, setEnd] = useState('');
+    const [userId, setUserId] = useState('');
+    const [users, setUsers] = useState([]);
+    
+    const options = {
+        method: 'GET',
+        headers: {
+            'Content-Type': 'application/json'
+        }
+    };
+
+    useEffect(() => {
+        fetch(`http://localhost:3000/sessions/${id}/edit`, options)
+        .then(res => res.json())
```

Comment:
Please handle the case where the res is not a JSON. Atleast check the res.statusCode === 200 before parsing it as a JSON